trdg/generators/from_dict.py

```diff
@@ -66,7 +66,5 @@ def __next__(self):
 
     def next(self):
         if self.generator.generated_count >= 999:
-            self.generator.strings = create_strings_from_dict(
-                self.length, self.allow_variable, 1000, self.dict
-            )
+            self.generator.strings = create_strings_from_dict(self.length, self.allow_variable, 1000, self.dict)
```

Comment:
This line is 112 characters long, did you format your code with black?

Reply:
That is strange:

    (core) ~/Projects/ocr/TextRecognitionDataGenerator $ black ./
    All done! ✨ 🍰 ✨
    15 files left unchanged.

Reply:
Alright, I can fix it afterward.

Reply:
Tried to set the black -l 88 (shoud be default) explicitly, hope this fixed the issue.

Reply:
It did, thanks!